### PR TITLE
add wait for JQuery and JS to JSExecutor

### PR DESCRIPTION
**Pull request by KonradFilipiak**
### Summary

*[Short description related to the code. More details about test scenarios should be kept 
in the JIRA ticket related to the pull request.]*

### Reference
JIRA Ticket:  *[Insert here a link to the JIRA ticket.]*

### Checklist
- [ ] Do your branch and PR name start with issue key from JirQA?
- [ ] Do your changes require to update the README.md file? Have you already done that?
- [ ] If you have changed the `initConfig.properties` file remember to update it in 
  * 1password (vault #QA), 
  * `gradle clean cucumber...` command in `.circleci/config.yml` file, 
  * environment variables in CircleCI project.
- [ ] Add the “Ready for review” label.
- [ ] Move the JIRA ticket to "Waiting for feedback".
- [ ] Remember to delete your branch after merging it.

### CODE REVIEW

File: src/main/java/managers/JSExecutor.java [hunk 1]
```diff
@@ -15,6 +15,19 @@ public JSExecutor(WebDriver driver) {
         js = (JavascriptExecutor) driver;
     }
 
+    public boolean waitForJQueryToLoad() {
+        try {
+            return (Long) executeScript("return jQuery.active") == 0;
+        } catch (Exception e) {
```
Comment (siksterka) [inline src/main/java/managers/JSExecutor.java]: I'm missing the wait part. This code is only checking if jQuery is active once. How do you plan to use this method?

Comment (KonradFilipiak) [reply src/main/java/managers/JSExecutor.java]: Obviously you're right, just fixed that

File: src/main/java/managers/JSExecutor.java [hunk 2]
```diff
@@ -15,6 +16,24 @@ public JSExecutor(WebDriver driver) {
...
+
+        try {
+            return Context.driverManager.wait.until(jQueryLoaded);
+        }catch (Exception e){
+            // no jQuery present
+            return true;
+        }
+    }
+
+    public boolean waitForJSToLoad() {
```
Comment (agamamla) [inline src/main/java/managers/JSExecutor.java]: similar method is in DriverManager class, waitForPageToLoad() - remove 1 to avoid duplicates

Comment (KonradFilipiak) [reply src/main/java/managers/JSExecutor.java]: one from DriverManager was removed

**Pull request merged**